splitgraph/core/image.py

```diff
@@ -166,7 +169,7 @@ def checkout(self, force: bool = False, layered: bool = False) -> None:
             self.object_engine.delete_table(target_schema, table)
 
         if layered:
-            self.lq_checkout()
+            self.lq_checkout(ddn_layout=ddn_layout)
```

Comment:
Since we've split the read and write shim schemas there's no use in having different overlay layout between cli (`_sgov_lower_my_table` foreign table, `my_table` view, `_sgov_upper_my_table` helper table) and ddn (`my_table` foreign table, `_sgov_merged_my_table` view, `_sgov_upper_my_table` helper table) now in theory.

Even if we decide to try and re-use those schemas to serve up some reads from them, we would need to keep track of the `dirty` tables. Since the target table is a dirty table (changed after executing the write), we would need to re-mount a clean shim schema anyway.